src/config.rs

```diff
@@ -695,10 +697,28 @@ fn config_from_env() -> Result<EnvConfig> {
         .ok()
         .and_then(|v| parse_size(&v));
 
+    let mut direct_mode_config = DirectModeConfig::default();
+    match env::var("SCCACHE_DIRECT").as_deref() {
+        Ok("on") | Ok("true") => direct_mode_config.use_direct_mode = true,
+        Ok("off") | Ok("false") => direct_mode_config.use_direct_mode = false,
+        _ => {}
+    };
+    // Disabling has precedence over enabling
+    match env::var("SCCACHE_NODIRECT").as_deref() {
+        Ok("on") | Ok("true") => direct_mode_config.use_direct_mode = false,
+        Ok("off") | Ok("false") => direct_mode_config.use_direct_mode = true,
+        _ => {}
+    };
+
+    if env::var("SCCACHE_DIRECT").is_ok() && env::var("SCCACHE_NODIRECT").is_ok() {
+        warn!("Both `SCCACHE_DIRECT` && `SCCACHE_NODIRECT` are defined");
```

Comment:
```suggestion
        warn!("Conflicting options: Both `SCCACHE_DIRECT` && `SCCACHE_NODIRECT` are defined");
```

Reply:
Imho we should prefer a single env var name and use its value.

Reply:
That works for me. I don't think accepted boolean values are unified across sccache. Usually (`1`, `on`, `true`, `0`, `off`, `false`), but maybe we want to be stricter (and complain when we don't get a valid value)?

Reply:
I think there was a clap extension / or impl. to handle the different variants which should be used for all of these features long term, but that'd be a breaking change. Adding new features / envs.